specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2019-03-01-preview/costmanagement.json

```diff
@@ -2618,6 +2663,54 @@
         }
       }
     },
+    "CheckEligibilityDefinition": {
+      "type": "object",
+      "description": "The check eligibility Connector credentials definition",
+      "properties": {
+        "kind": {
+          "type": "string",
+          "description": "Connector kind (eg aws)"
```

Comment:
Should "aws" be capitalized here? ("AWS")

Reply:
"aws" is already stored in the Db from all the cloud connectors apis. Hence, I would want to keep it lowercase